[INFO][2024-08-09 01:38:10,946][autoreload.py:677]Watching for file changes with StatReloader
[INFO][2024-08-09 03:40:07,994][middleware.py:13]Received GET request to / from AnonymousUser with query params: <QueryDict: {}> and POST data: <QueryDict: {}>
[INFO][2024-08-09 11:46:25,896][middleware.py:13]Received GET request to / from AnonymousUser with query params: <QueryDict: {}> and POST data: <QueryDict: {}>
[INFO][2024-08-09 15:26:59,882][autoreload.py:266]/www/data/task/views.py changed, reloading.
[INFO][2024-08-09 15:27:00,342][autoreload.py:677]Watching for file changes with StatReloader
[INFO][2024-08-09 16:14:46,102][autoreload.py:266]/www/data/task/views.py changed, reloading.
[INFO][2024-08-09 16:14:46,527][autoreload.py:677]Watching for file changes with StatReloader
[INFO][2024-08-09 16:15:30,814][middleware.py:13]Received PUT request to /api/task/manage/ from AnonymousUser with query params: <QueryDict: {}> and POST data: <QueryDict: {}>
[INFO][2024-08-09 16:19:11,543][autoreload.py:266]/www/data/task/views.py changed, reloading.
[INFO][2024-08-09 16:19:12,011][autoreload.py:677]Watching for file changes with StatReloader
[INFO][2024-08-09 16:19:19,633][middleware.py:13]Received PUT request to /api/task/manage/ from AnonymousUser with query params: <QueryDict: {}> and POST data: <QueryDict: {}>
[WARNING][2024-08-09 16:19:19,695][views.py:67]修改失败，任务不存在
[INFO][2024-08-09 16:19:39,416][middleware.py:13]Received PUT request to /api/task/manage/ from AnonymousUser with query params: <QueryDict: {}> and POST data: <QueryDict: {}>
[ERROR][2024-08-09 16:19:39,474][views.py:122]publish error
[ERROR][2024-08-09 16:19:39,509][log.py:241]Internal Server Error: /api/task/manage/
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/views/generic/base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/usr/local/lib/python3.11/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/usr/local/lib/python3.11/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/www/data/task/views.py", line 77, in put
    err = self.pulish(data["content"])
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/www/data/task/views.py", line 123, in pulish
    raise e
  File "/www/data/task/views.py", line 111, in pulish
    content = json.loads(content)
              ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[INFO][2024-08-09 16:20:42,142][autoreload.py:266]/www/data/task/views.py changed, reloading.
[INFO][2024-08-09 16:20:42,576][autoreload.py:677]Watching for file changes with StatReloader
[INFO][2024-08-09 16:20:43,930][middleware.py:13]Received PUT request to /api/task/manage/ from AnonymousUser with query params: <QueryDict: {}> and POST data: <QueryDict: {}>
[ERROR][2024-08-09 16:20:43,996][views.py:126]publish error
[ERROR][2024-08-09 16:20:43,997][views.py:82]Expecting value: line 1 column 1 (char 0)
[ERROR][2024-08-09 16:20:44,034][log.py:241]Internal Server Error: /api/task/manage/
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/rest_framework/response.py", line 74, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/rest_framework/renderers.py", line 99, in render
    ret = json.dumps(
          ^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/rest_framework/utils/json.py", line 25, in dumps
    return json.dumps(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/json/__init__.py", line 238, in dumps
    **kw).encode(obj)
          ^^^^^^^^^^^
  File "/usr/local/lib/python3.11/json/encoder.py", line 200, in encode
    chunks = self.iterencode(o, _one_shot=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/json/encoder.py", line 258, in iterencode
    return _iterencode(o, 0)
           ^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/rest_framework/utils/encoders.py", line 67, in default
    return super().default(obj)
           ^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/json/encoder.py", line 180, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type JSONDecodeError is not JSON serializable
[INFO][2024-08-09 16:21:27,846][autoreload.py:266]/www/data/task/views.py changed, reloading.
[INFO][2024-08-09 16:21:28,284][autoreload.py:677]Watching for file changes with StatReloader
[INFO][2024-08-09 16:21:29,004][middleware.py:13]Received PUT request to /api/task/manage/ from AnonymousUser with query params: <QueryDict: {}> and POST data: <QueryDict: {}>
[ERROR][2024-08-09 16:21:29,093][views.py:126]publish error
[ERROR][2024-08-09 16:21:29,093][views.py:82]Expecting value: line 1 column 1 (char 0)
[ERROR][2024-08-09 16:21:29,121][log.py:241]Internal Server Error: /api/task/manage/
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/rest_framework/response.py", line 74, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/rest_framework/renderers.py", line 99, in render
    ret = json.dumps(
          ^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/rest_framework/utils/json.py", line 25, in dumps
    return json.dumps(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/json/__init__.py", line 238, in dumps
    **kw).encode(obj)
          ^^^^^^^^^^^
  File "/usr/local/lib/python3.11/json/encoder.py", line 200, in encode
    chunks = self.iterencode(o, _one_shot=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/json/encoder.py", line 258, in iterencode
    return _iterencode(o, 0)
           ^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/rest_framework/utils/encoders.py", line 67, in default
    return super().default(obj)
           ^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/json/encoder.py", line 180, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type JSONDecodeError is not JSON serializable
[INFO][2024-08-09 16:24:10,555][autoreload.py:266]/www/data/task/views.py changed, reloading.
[INFO][2024-08-09 16:24:10,937][autoreload.py:677]Watching for file changes with StatReloader
[INFO][2024-08-09 16:24:12,306][middleware.py:13]Received PUT request to /api/task/manage/ from AnonymousUser with query params: <QueryDict: {}> and POST data: <QueryDict: {}>
[ERROR][2024-08-09 16:24:12,375][views.py:124]Expecting value: line 1 column 1 (char 0)
[INFO][2024-08-09 16:24:41,177][middleware.py:13]Received PUT request to /api/task/manage/ from AnonymousUser with query params: <QueryDict: {}> and POST data: <QueryDict: {}>
[ERROR][2024-08-09 16:24:41,235][views.py:124]Expecting value: line 1 column 1 (char 0)
[INFO][2024-08-09 16:24:46,899][middleware.py:13]Received GET request to /api/task/manage/ from AnonymousUser with query params: <QueryDict: {'task_id': ['2']}> and POST data: <QueryDict: {}>
[INFO][2024-08-09 16:25:38,974][autoreload.py:266]/www/data/task/views.py changed, reloading.
[INFO][2024-08-09 16:25:39,365][autoreload.py:677]Watching for file changes with StatReloader
[INFO][2024-08-09 16:25:40,173][middleware.py:13]Received GET request to /api/task/manage/ from AnonymousUser with query params: <QueryDict: {'task_id': ['2']}> and POST data: <QueryDict: {}>
[INFO][2024-08-09 16:25:45,472][middleware.py:13]Received PUT request to /api/task/manage/ from AnonymousUser with query params: <QueryDict: {}> and POST data: <QueryDict: {}>
[ERROR][2024-08-09 16:25:45,526][views.py:124]Expecting value: line 1 column 1 (char 0)
[ERROR][2024-08-09 16:25:45,527][views.py:80]Expecting value: line 1 column 1 (char 0)
[INFO][2024-08-09 16:25:54,022][middleware.py:13]Received PUT request to /api/task/manage/ from AnonymousUser with query params: <QueryDict: {}> and POST data: <QueryDict: {}>
[ERROR][2024-08-09 16:25:54,077][views.py:124]Expecting value: line 1 column 1 (char 0)
[ERROR][2024-08-09 16:25:54,077][views.py:80]Expecting value: line 1 column 1 (char 0)
[INFO][2024-08-09 16:26:24,661][autoreload.py:266]/www/data/task/views.py changed, reloading.
[INFO][2024-08-09 16:26:25,046][autoreload.py:677]Watching for file changes with StatReloader
[INFO][2024-08-09 16:26:26,372][middleware.py:13]Received PUT request to /api/task/manage/ from AnonymousUser with query params: <QueryDict: {}> and POST data: <QueryDict: {}>
[ERROR][2024-08-09 16:26:26,454][views.py:124]Expecting value: line 1 column 1 (char 0)
[ERROR][2024-08-09 16:26:26,455][views.py:80]json解析错误
[INFO][2024-08-09 16:26:46,596][middleware.py:13]Received PUT request to /api/task/manage/ from AnonymousUser with query params: <QueryDict: {}> and POST data: <QueryDict: {}>
[WARNING][2024-08-09 16:26:46,644][views.py:67]修改失败，任务不存在
[INFO][2024-08-09 16:26:51,540][middleware.py:13]Received PUT request to /api/task/manage/ from AnonymousUser with query params: <QueryDict: {}> and POST data: <QueryDict: {}>
[INFO][2024-08-09 16:27:34,215][middleware.py:13]Received PUT request to /api/task/manage/ from AnonymousUser with query params: <QueryDict: {}> and POST data: <QueryDict: {}>
[ERROR][2024-08-09 16:27:34,268][views.py:124]the JSON object must be str, bytes or bytearray, not dict
[ERROR][2024-08-09 16:27:34,268][views.py:80]json解析错误
[INFO][2024-08-09 16:32:25,545][autoreload.py:266]/www/data/task/views.py changed, reloading.
[INFO][2024-08-09 16:32:26,011][autoreload.py:677]Watching for file changes with StatReloader
[INFO][2024-08-09 16:32:29,453][middleware.py:13]Received PUT request to /api/task/manage/ from AnonymousUser with query params: <QueryDict: {}> and POST data: <QueryDict: {}>
[INFO][2024-08-09 16:32:29,516][views.py:77]<class 'dict'>
[ERROR][2024-08-09 16:32:29,516][views.py:125]the JSON object must be str, bytes or bytearray, not dict
[ERROR][2024-08-09 16:32:29,516][views.py:81]json解析错误
[INFO][2024-08-09 16:34:17,498][autoreload.py:266]/www/data/task/views.py changed, reloading.
[INFO][2024-08-09 16:34:17,874][autoreload.py:677]Watching for file changes with StatReloader
[INFO][2024-08-09 16:34:19,233][middleware.py:13]Received PUT request to /api/task/manage/ from AnonymousUser with query params: <QueryDict: {}> and POST data: <QueryDict: {}>
[INFO][2024-08-09 16:34:19,298][views.py:121]{'title': '第一天的任务1', 'status': 'done', 'desc': '你好！这里是任务简介，可以在这里添加任务的简单描述！', 'content': '<h1 style="text-indent: 0em !important;" id="h-27250198">智慧笔匠 - 任务时间轴</h1><p style="text-indent: 0em !important;">你好！这里是智慧笔匠的任务时间轴，我可以帮助你安排并管理工作日程！</p><p style="text-indent: 0em !important;"></p><p style="text-indent: 0em !important;">你可以点击上方的日程栏切换每一天的任务项，并在下方修改您的任务标题、任务状态、任务简介和任务详情。</p><p style="text-indent: 0em !important;"></p><p style="text-indent: 0em !important;">你也可以使用AI助手自动规划工作内容并安排工作时间。</p><ai-plan2></ai-plan2><p style="text-indent: 0em !important;"></p>', 'id': '3e63d4c2-0b23-49eb-ad48-bf3bc050f6c3'}
[INFO][2024-08-09 16:34:19,298][views.py:121]{'title': '第一天的任务2', 'status': 'done', 'desc': '第一天要做的第二件事...', 'content': '<h1 style="text-indent: 0em !important;" id="h-b4506869">这里可以编写任务详情</h1><p style="text-indent: 0em !important;"></p>', 'id': 'e50ec2c2-59d7-465e-a241-3c398d5d655a'}
[INFO][2024-08-09 16:34:19,298][views.py:121]{'id': '70567d88-f82f-4d9d-b845-f18e6bcd0117', 'title': '第二天的任务', 'status': 'done', 'desc': '第二天要做的工作简介', 'content': '<h1 style="text-indent: 0em !important;" id="h-037759ce">第二天的工作内容</h1><p style="text-indent: 0em !important;">...</p>'}
[INFO][2024-08-09 16:34:19,298][views.py:121]{'id': '99455f69-8f9c-4e14-be73-cc7db1d19404', 'title': '第三天的任务1', 'status': 'ongoing', 'desc': '第三天要做的第一件事的工作简介', 'content': '<h1 style="text-indent: 0em !important;" id="h-037e3f96">第三天的第一项任务的具体内容</h1><p style="text-indent: 0em !important;"></p>'}
[INFO][2024-08-09 16:34:19,299][views.py:121]{'id': '7add1a7f-42a3-4099-a4b2-7686b6e1e608', 'title': '第三天的任务2', 'status': 'pending', 'desc': '第三天要做的第二件事的工作简介', 'content': '<h1 style="text-indent: 0em !important;" id="h-1149b678">第三天的第二项任务的具体内容</h1><p style="text-indent: 0em !important;"></p>'}
[INFO][2024-08-09 16:34:19,299][views.py:121]{'id': '7addb56c-f359-4d37-a21a-328dafaaf2b5', 'title': '第四天的任务', 'status': 'prime', 'desc': '这里是第四天的任务简介', 'content': '<h1 style="text-indent: 0em !important;" id="h-c5e15f4b">这里是第四天的任务内容</h1><p style="text-indent: 0em !important;"></p>'}
[INFO][2024-08-09 16:34:19,299][views.py:121]{'id': 'c3da8f02-bf4f-4dbe-956d-5774d8470664', 'title': '第五天的任务', 'status': 'prime', 'desc': '这里是第五天的任务简介', 'content': '<h1 style="text-indent: 0em !important;" id="h-34863faa">这里是第五天的任务内容</h1><p style="text-indent: 0em !important;"></p>'}
[INFO][2024-08-09 16:34:28,770][middleware.py:13]Received GET request to /api/task/manage/ from AnonymousUser with query params: <QueryDict: {'task_id': ['2']}> and POST data: <QueryDict: {}>
[INFO][2024-08-09 17:52:46,213][autoreload.py:266]/www/data/AI/views.py changed, reloading.
[INFO][2024-08-09 17:52:46,634][autoreload.py:677]Watching for file changes with StatReloader
[INFO][2024-08-09 18:06:46,126][autoreload.py:266]/www/data/AI/views.py changed, reloading.
[INFO][2024-08-09 18:06:46,516][autoreload.py:677]Watching for file changes with StatReloader
[INFO][2024-08-09 18:11:15,216][middleware.py:13]Received GET request to /media/avatar/3_2_ML90heW.jpg from AnonymousUser with query params: <QueryDict: {}> and POST data: <QueryDict: {}>
[INFO][2024-08-09 18:11:15,254][middleware.py:13]Received GET request to /api/user/userinfo/ from AnonymousUser with query params: <QueryDict: {}> and POST data: <QueryDict: {}>
[INFO][2024-08-09 18:11:37,082][middleware.py:13]Received POST request to /api/ai/mysystem/ from AnonymousUser with query params: <QueryDict: {}> and POST data: <QueryDict: {}>
[INFO][2024-08-09 18:13:48,978][autoreload.py:266]/www/data/AI/views.py changed, reloading.
[INFO][2024-08-09 18:13:49,401][autoreload.py:677]Watching for file changes with StatReloader
[INFO][2024-08-09 18:18:24,157][middleware.py:13]Received POST request to /api/ai/mysystem/ from AnonymousUser with query params: <QueryDict: {}> and POST data: <QueryDict: {}>
[WARNING][2024-08-09 18:19:14,262][views.py:509]Empty response
